README.md

```diff
@@ -110,10 +121,10 @@ All [puppet metaparameters](https://docs.puppetlabs.com/references/latest/metapa
 
 ## Limitations
 
-- DSC Composite Resources are not yet fully supported.
+- DSC Composite Resources are not supported.
 - PSObjects like 'PSCredential' as parameters value not yet supported.
 
-## Known Issues
+### Known Issues
 
 When installing the module on Windows you may run into an issue regarding long file names (LFN), due to the long paths of generated schema files. If you install your module on a Linux master and then use plugin sync, you will not likely see this issue, but if you are attempting to install the module on a Windows machine, you may run into an error that looks similar to the following during `puppet module install puppetlabs-dsc`:
```

Comment:
Commas, rearranged some parts, run on sentences.

When installing the module on Windows you may run into an issue regarding long file names (LFN) due to the long paths of the generated schema files. If you install your module on a Linux master and then use plugin sync you will  likely not see this issue. If you are attempting to install the module on a Windows machine using `puppet module install puppetlabs-dsc` you may run into an error that looks similar to the following: